README.md

```diff
@@ -1,5 +1,42 @@
 # Create PDF/A-4 Conformance Documents using C#
-This repository contains Syncfusion PDF library examples that explains how to create PDF/A-4 conformance and convert normal PDF to PDF/A-4 conformance document.
 
-## Blog reference
-[Create PDF/A-4 Documents for Long-Term Archiving in C#](https://www.syncfusion.com/blogs/post/create-pdf-a-4-documents-for-long-term-archiving-in-c.aspx)
+This repository contains [Syncfusion PDF Library](https://www.syncfusion.com/document-processing/pdf-framework/net) examples that explains how to create PDF/A-4 conformance and convert normal PDF to PDF/A-4 conformance document.
```

Comment:
This repository contains examples of creating PDF/A-4 conformance documents and converting PDF to PDF/A-4 document.

Reply:
changes Updated